lib/index.js

```diff
@@ -87,6 +87,22 @@ function _createUrl(config) {
   return url;
 }
 
+function _sanitizeUrl(url) {
+  const urlParts = require('url').parse(url);
+  let user;
+  if(urlParts.auth) {
+    user = urlParts.auth.split(':')[0];
+  }
+  let sanitizedUrl;
+  if(user) {
+    sanitizedUrl =
+      `${urlParts.protocol}//${user}:****@${urlParts.host}${urlParts.path}`;
```

Comment:
my preference would be that we don't even log the user, @JSAssassin @aljones15 @davidlehn @dlongley @gannan08 thoughts?

Reply:
Which would also simply the implementation somewhat here.

Reply:
In a production deployment, I do not see the added value for logging the user.

Reply:
@mattcollier @gannan08 agreed. I can update that.

Reply:
NODE_ENV values could also be used to control this if it turns out more debug info would be useful.

Reply:
-1 to bringing NODE_ENV into this, at least at this time.